katas/es6/language/array-api/fill.js

```diff
@@ -35,4 +35,22 @@ describe('`Array.prototype.fill` can fill up an array with one value', () => {
     assert.deepEqual(arr, [1, 42, 3]);
   });
 
+
+  it('is a mutable method, and will change this object itself', function() {
```

Comment:
```suggestion
  it('is a mutating method, and will change this object itself', function() {
```

isn't that the right term?